samples/peripheral/radio_test/Kconfig

```diff
@@ -14,5 +14,15 @@ config RADIO_TEST_USB
 	  Use USB instead of UART as the Radio Test shell transport.
 	  For nRF5340 the USB from application core is used as communication interface.
 
+config RADIO_TEST_POWER_CONTROL_AUTOMATIC
```

Comment:
If this is set to "n" then the FEM power model is not needed, and `CONFIG_MPSL_FEM_POWER_MODEL` could be set to "n".
Now it is set to "y" by default (if only the needed hardware is available), which does not affect anything functionally, but performs a lot of unneeded operations in the background and wastes space.
However, I'm not sure if there is an elegant way to turn a symbol off for a non-default value of the `RADIO_TEST_POWER_CONTROL_AUTOMATIC` option, so this is just something to consider,

Reply:
I think that user can disable it himself because these config only enables the splitter functionality in the Radio Test.
I would like to avoid redefinition of this symbol here, it seems to me that it is the only way to disable auto enabled option